### ISSUE TITLE
TOKEN ENDPOINT values

### ISSUE THREAD

**Issue by arbor95** [Surfoo/oauth2-geocaching#5]
the geocaching API, Paragraph TOKEN ENDPOINT tells about the response as a JSON construct with the four parts "access_token", "token_type", "expires_in" and "refresh_token".
How can I get the values for this?

**Comment by Surfoo** [comment]
Hello @arbor95 

Did you check the demo file? Here an example: https://github.com/Surfoo/oauth2-geocaching/blob/develop/demo/index.php#L64

You have also `getRefreshToken()`, `getExpires()`, `hasExpired()` in the source code of oauth2-client: https://github.com/thephpleague/oauth2-client/blob/master/src/Token/AccessTokenInterface.php

**Comment by arbor95** [comment]
Thx for answer. Yes, the demo file is that, what I extended.

Have a look at https://api.groundspeak.com/documentation#authorization , subpart TOKEN ENDPOINT , Response Values .

These Response Values are, what i need.

Digging into the thephpleague (AccessToken.php) I see, what I did not expect:
1. getValues() returns only the non-standard values (in this case only the 'token_type' (for gc is always 'bearer')
2. I did not expect the 'access_token' to be that number of bytes long (more than 900)
3. the 'expires-in' converted into date

So again thx for your help
